### PR TITLE
feat: add context directory module

### PR DESCRIPTION
**Pull request by TomChv**

### CODE REVIEW

File: context-dir/src/main.go
Comment (jedevc) [inline context-dir/src/main.go]: Can we also get some sub-paths? Something like `/data` and `./data`, so we can check the differences between those as well?

**Review by jedevc — COMMENTED** [review]
LGTM, cc @grouville - can we also mirror these changes when merged to the other repos we have?

File: context-dir/src/main.go
Comment (TomChv) [reply context-dir/src/main.go]: I mean, `/` will point to the root of the repo while `.` will point to the current dir so we can already do some further queries in our test to get a specific dir

like `dagger call absolute-path directory --path "context-dir"` & `dagger call relative-path directory --path "src"`

Comment (TomChv) [reply context-dir/src/main.go]: So should I still add new method?

Comment (jedevc) [reply context-dir/src/main.go]: I think it would be good to have a couple of new methods, and put some content there, just so we can check that the path joining logic is all correct in the tests.

Comment (TomChv) [reply context-dir/src/main.go]: I see, alright

Comment (TomChv) [reply context-dir/src/main.go]: Changes done

**Comment by TomChv** [comment]
@jedevc Still waiting for your approval on that one :D

**Review by jedevc — APPROVED** [review]

**Pull request merged**
